packages/amplify-cli/src/init-steps/preInitSetup.ts

```diff
@@ -22,6 +35,38 @@ export const preInitSetup = async (context: $TSContext): Promise<$TSContext> =>
   return context;
 };
 
+/**
+ * recommend using Gen 2 or continue with Gen 1.
+ * ask for why they are using Gen 1 and store the answer in project-config
+ */
+export const gen2Recommendation = async (context: $TSContext): Promise<$TSContext> => {
```

Comment:
I think it doesn't need to be exported anymore?

Reply:
We need to export it for the unit test

Reply:
reverted the export 
1e8e25b